elide-integration-tests/src/test/java/com/yahoo/elide/async/integration/tests/AsyncIT.java

```diff
@@ -127,7 +126,9 @@ public void jsonApiRequestTests() throws InterruptedException {
                                         attributes(
                                                 attr("query", "/book?sort=genre&fields%5Bbook%5D=title"),
                                                 attr("queryType", "JSONAPI_V1_0"),
-                                                attr("status", "QUEUED")
+                                                attr("status", "QUEUED"),
+                                                attr("requestId", "1001"),
+                                                attr("asyncAfterSeconds", "10")
```

Comment:
I think we should restructure these tests into two kinds:
1) One that sets asyncAfter to 0 - and does the traditional async loop below.
2) One that sets asyncAfter to 10 - and then checks the immediate returned object (it should contain the completed result).